### ISSUE TITLE
Why have less than/greater than?

### ISSUE THREAD

**Issue by wbinnssmith** [aaivazis/redux-responsive#8]
Hey!

First of all, love this idea. It fits perfectly with the redux philosophy of a centralized state store and the browser state totally belongs there. Super useful.

I'm maintaining a separate version without the less than/greater than options since doing a quick check like

```js
if (state.width < sizes.SMALL) {
   ...
}
```
is equally as expressive and less coupled to this implementation as 
```js
if (state.lessThan.small) {
   ...
}
```

Does this implementation, along with the complexity of a reducer factory, provide any more value than simply storing breakpoints in a plain old js object mapping? 

Thanks again for the module!

**Comment by aaivazis** [comment]
Hey there,

Thanks for the kind words. I'm glad other people are liking this little module. Also, sorry it took me a bit to reply.

I agree that they are very similar semantically, and your alternative probably has slightly better performance (constant response time rather than depending on the number of breakpoints) but I'm not sure anyone uses enough breakpoints for that to matter.

My original design for the reducer was to be completely contained and not rely on a comparison like you suggest for a few different reasons. For one, `browser.lessThan.small` does not rely on importing an external breakpoint map. Every time you use the reducer you would need to import the breakpoints which seems like a lot of repetition and I would most likely forget it a few times. By moving the breakpoints into the reducer, I don't have to worry about what it means for the browser to be `medium`, I just ask the reducer if its true. I prefer this to have to remember that the breakpoints are stored in a particular object somewhere in my filesystem.

tl;dr - the current implementation is fewer words/loc and requires a smaller mental model on the part of the developer which makes it easier for me to work with.

**Comment by aaivazis** [comment]
maybe @montemishkin has some input?

**Comment by montemishkin** [comment]
I agree with @aaivazis: the current implementation keeps things simple and minimizes the number of imports, etc that the user needs to juggle.  

Also, it seems @wbinnssmith's idea is still fully possible with the current api (by using `state.width` and just ignoring the `lessThan` and `greaterThan` fields).

Lastly, I also agree with @aaivazis that I'm glad you are enjoying the module, and I appreciate your input!

**Comment by aaivazis** [comment]
Thanks for the input @montemishkin. I'm going to go ahead and close this issue to keep things tidy.

@wbinnssmith: Feel free to open it if you have more to add

**Comment by wbinnssmith** [comment]
Sounds good. Thanks again!
